app/src/main/res/layout/activity_main.xml

```diff
@@ -11,89 +11,99 @@
-        app:layout_constraintTop_toBottomOf="@id/temperature"
         android:text="@string/feels_like"
-    />
+        android:textColor="@color/black"
+        app:layout_constraintEnd_toEndOf="@id/temperature"
+        app:layout_constraintStart_toStartOf="@id/temperature"
+        app:layout_constraintTop_toBottomOf="@id/temperature" />
+
     <TextView
         android:id="@+id/low"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textSize="18sp"
-        android:textColor="@color/black"
         android:layout_marginTop="24dp"
-        app:layout_constraintStart_toStartOf="@id/temperature"
-        app:layout_constraintTop_toBottomOf="@id/feels_like"
         android:text="@string/low"
-        />
+        android:textColor="@color/black"
+        android:textSize="18sp"
+        app:layout_constraintStart_toStartOf="@id/temperature"
+        app:layout_constraintTop_toBottomOf="@id/feels_like" />
+
     <TextView
         android:id="@+id/high"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textSize="18sp"
+        android:text="@string/high"
         android:textColor="@color/black"
+        android:textSize="18sp"
         app:layout_constraintStart_toStartOf="@id/low"
-        app:layout_constraintTop_toBottomOf="@id/low"
-        android:text="@string/high"
-        />
+        app:layout_constraintTop_toBottomOf="@id/low" />
+
     <TextView
         android:id="@+id/humidity"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textSize="18sp"
+        android:text="@string/humidity"
         android:textColor="@color/black"
+        android:textSize="18sp"
         app:layout_constraintStart_toStartOf="@id/low"
-        app:layout_constraintTop_toBottomOf="@id/high"
-        android:text="@string/humidity"
-        />
+        app:layout_constraintTop_toBottomOf="@id/high" />
+
     <TextView
         android:id="@+id/pressure"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:textSize="18sp"
+        android:text="@string/pressure"
         android:textColor="@color/black"
+        android:textSize="18sp"
         app:layout_constraintStart_toStartOf="@id/low"
-        app:layout_constraintTop_toBottomOf="@id/humidity"
-        android:text="@string/pressure"
-        />
+        app:layout_constraintTop_toBottomOf="@id/humidity" />
+    <Button
+        android:id="@+id/forecast_button"
+        android:layout_width="128dp"
+        android:layout_height="48dp"
+        android:text="@string/forecast"
+        app:layout_constraintStart_toStartOf="@+id/city_name"
+        app:layout_constraintTop_toBottomOf="@id/pressure"
+        android:layout_marginTop="12dp"
+        android:backgroundTint="#AAAAAA"
+        android:textColor="@color/black"
+        android:onClick="switchActivity"/>
```

Comment:
This works, but isn't best practice. It merges declarative UI (i.e. XML) with view logic. In the future use `setOnClickListener` in your Activities and Fragments.